### ISSUE TITLE
restifyError event is not documented

### ISSUE THREAD

**Issue by dotchev** [restify/node-restify#1358]
- [x] Used appropriate template for the issue type
- [x] Searched both open and closed issues for duplicates of this issue
- [x] Title adequately and _concisely_ reflects the feature or the bug

# Bug Report

## Expected behaviour

All restify events are documented.

## Actual behaviour

'restifyError' event is emited in the code
https://github.com/restify/node-restify/blob/5.x/lib/server.js#L933

But it is not mentioned in the documentation on http://restify.com/

## Are you willing and able to fix this?

Yes, but I am not sure about the exact behavior.

**Comment by retrohacker** [comment]
Hey @dotchev!

Good catch! The documentation for 5.x is going through some changes right now and we will be shipping a new website in the near future. Unfortunately I don't believe there is a single definitive place to point people to contributing doc fixes to at the moment since things are in flux. Hoping to get this all unblocked in the very near future and then I'll circle back to this issue :heart:

**Comment by hekike** [comment]
It's fixed now by the automated docs generation
